app/src/main/java/com/sun/heartrate/ui/main/MainPagerAdapter.kt

```diff
@@ -0,0 +1,26 @@
+package com.sun.heartrate.ui.main
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentPagerAdapter
+
+
+class MainPagerAdapter(
+    fragmentManager: FragmentManager?
+) : FragmentPagerAdapter(fragmentManager, BEHAVIOR_RESUME_ONLY_CURRENT_FRAGMENT) {
+    
+    private val fragments = mutableListOf<Fragment>()
```

Comment:
```suggestion
    private val fragments = listOf<Fragment>(...fragment...)
```

Reply:
S lai la list nhi?

Reply:
Đoạn này anh Toản code đó @tr4n, không set cứng các fragment đâu mà là thêm vào bằng phương thức addFragment bên dưới.